glean-core/python/glean/glean.py

```diff
@@ -219,5 +240,60 @@ def get_data_dir(cls) -> Path:
         """
         return cls._data_dir
 
+    @classmethod
+    def test_collect(cls, ping: "PingType") -> str:
+        """
+        Collect a ping and return as a string.
+        """
+        return _ffi.ffi_decode_string(
+            _ffi.lib.glean_ping_collect(cls._handle, ping._handle)
+        )
+
+    @classmethod
+    def send_pings(cls, pings: List["PingType"]):
```

Comment:
Is there any way we could flag/make this private or only accessible from within this package?

Reply:
You can't really make anything private in Python, but we can flag it with a `_` prefix.  I missed these were internal in the Kotlin implementation.